src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -984,13 +984,14 @@ public async Task SendAsync_ReadFromSlowStreamingServer_PartialDataReturned()
         }
 
         [Theory]
-        [InlineData(true, true)]
-        [InlineData(true, false)]
-        [InlineData(false, true)]
-        [InlineData(false, false)]
-        [InlineData(null, false)]
+        [InlineData(true, true, true)]
+        [InlineData(true, true, false)]
+        [InlineData(true, false, false)]
```

Comment:
Why we're not testing slow chunks when `chunked` == true and `enableWasmStreaming` == false? As far as I can see, usage of `slowChunks` 
https://github.com/dotnet/runtime/blob/0516f2cd7d1051a8c9db05dcd35e2536e0fb5dd0/src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs#L1197
depends only on `chunked` and it makes sense to test slow chunks only when `chunked` is true. For `enableWasmStreaming` we don't have such a relation, it makes sense to test slow chunks for all values of `enableWasmStreaming`, doesn't it?

Reply:
thanks, `enableWasmStreaming` makes no sense outside of browser. I will improve it.

Reply:
We established that in this PR we are not planning to handle the case `[InlineData(true, false, true)]`, even though technically such a case is possible for coreCLR.